templates/ezplatform.yaml

```diff
@@ -0,0 +1,37 @@
+# The schema is versioned so that we can establish code paths differently in the future if we need to change this.
+version: 1
+
+# Templates are a small amount of information supporting a template URL. Each template is selectable at the project-creation step.
+info:
+    # Unique machine name, prefaced by a vendor or organization identifier
+    id: ezsystems/ezplatform
+    # The human-readable name of the template.
+    name: eZ Platform 2.5.5
+    # Human-readable descriptive text for the template. Supports limited HTML.
+    description: The latest version of <b>eZ Platform</b> without customization. Perfect to start a new project.
+    # A list of tags associated with the template.
+    tags:
+        - PHP
+        - eZ
+        - CMS
+        - Symfony
```

Comment:
```suggestion
```

Reply:
Thanks @Crell 
I do have a question about this suggestion. I can see that `Akeneo` and `Pimcore` listed with Symfony. `eZ Platform` is also using Symfony fullstack. Is there a rule to be able to have `Symfony`?

Reply:
Hm, you're right, we are tagging other Symfony-based apps.  I thought we weren't.  Ignore this one.